install.sh

```diff
@@ -49,8 +50,17 @@ while [[ "$#" -ne '0' ]] ; do
 			HOSTNAME_PARAM="${1#hostname=}"
 			shift
 			;;
+		net)
+		        shift
+			NET_PARAM="$1"
+			shift
+			;;
+		net=*)
+			NET_PARAM="${1#net=}"
+			shift
+			;;
 		net-host)
-			NET_HOST_PARAM=true
+			NET_PARAM="host"
```

Comment:
this is not fail-safe since the user could still provide conflicting net settings (`net-host` and `net mylan`)

Reply:
No problem, let the last one win.